.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/setup
+
+      - name: "Show the Plan"
+        run: echo "$(node ./test-packages/release/src/cli.js explain-plan)"
```

Comment:
It shows the plan in the CI logs (explain plan), so we have a record of what it attempted.
Without this, we don't have any way to verify what it thinks it's doing, as trusting files outside of "our current line of sight", isn't reliable, because mistakes are too easy to make -- it's best to have the debug information readily available.

Reply:
I think you might be missing my point. The design of the release process is that you should prepare a plan and commit that plan as a file to a PR and merge to main. This is rebuilding the plan from scratch instead of reading from the already prepared release plan.

Reply:
Hm, maybe the description of these needs to be updated

![image](https://github.com/embroider-build/embroider/assets/199018/79142940-1475-4c09-8710-d2cc3b9c4826)

I'll change to only catting the file then